presentation.ipynb

```diff
@@ -465,7 +467,9 @@
     "NumPy currently targets computation involving:\n",
     "\n",
     " * in-memory, homogenously-typed array data\n",
-    " * cpu-based"
+    " * cpu-based\n",
+    " \n",
+    "but provides protocols to allow developers of packages targeting other computational models hooks to seamlessly interoperate in Python."
```

Comment:
like above, not sure this is needed here

Reply:
I think you're right that there should be some sort of explicit mention of the extensibility here, as it is an important feature (just maybe not one that *users* are very familiar with).